src/client/Ruby/ms-rest/lib/ms_rest/http_operation_request.rb

```diff
@@ -121,7 +117,20 @@ def to_json(*a)
         log: log  
       }.to_json(*a)
     end
-    
+
+    def self.create_faraday_connection(base_uri, options = {})
+      Faraday.new(:url => base_uri, :ssl => MsRest.ssl_options) do |faraday|
```

Comment:
Should we also streamline how `ssl_options` are getting passed to the `create_faraday_connection`. Example: options should allow setting them or not?